enacts/enactstozarr.py

```diff
@@ -48,53 +36,45 @@ def set_up_dims(xda):
     # and look into xESMF.
     #
     # [1] https://climatedataguide.ucar.edu/climate-data-tools-and-analysis/regridding-overview
-    rr_mrg = rr_mrg.interp(
-        X=np.arange(rr_mrg.X.min(), rr_mrg.X.max() + RESOLUTION, RESOLUTION),
-        Y=np.arange(rr_mrg.Y.min(), rr_mrg.Y.max() + RESOLUTION, RESOLUTION),
-    )
-
-rr_mrg = rr_mrg.chunk(chunks=CHUNKS)
-
-xr.Dataset().merge(rr_mrg).to_zarr(
-  store = RR_MRG_ZARR_PATH
-)
-
-if not os.access(CONFIG["data_dir"], os.W_OK | os.X_OK):
-    sys.exit("can't write to output directory")
-
-def set_up_dims(xda):
-    datestr = Path(xda.encoding["source"]).name.split("_")[2]
-    year = int(datestr[0:4])
-    month = int(datestr[4:6])
-    dekad = int(datestr[6:7])
-    day = (dekad - 1) * 10 + 1
-    xda = xda.expand_dims(T = [dt.datetime(year, month, day)])
```

Comment:
AH! Actually: this is the dekad data case... that's why there were 2 set_up_dims... One came from enactstozarr working on daily data, the other from dekadal...

:(

Sorry Drew, I am afraid you are going to have to reconcile this. Maybe 2 functions: set_up_daily_dims and set_up_dekad_dims and a new argument to convert to tell which one to use?

Then you probably want the config files that use these to be set up like you're setting up the daily data. I think only kmd and zmd have it. And then monthly/maproom.py and layout.py will have to use the right CONFIG info...

Sorry I didn't see that earlier...

Reply:
Okay, yes in this case I can resolve the other things and then maybe you can take over this part.